.clang-format

```diff
@@ -1,101 +1,209 @@
+BinPackArguments: true
 BinPackParameters: false
+BitFieldColonSpacing: Both
 BraceWrapping:
-  AfterClass:      true
-  AfterControlStatement: false
-  AfterEnum:       false
-  AfterFunction:   true
-  AfterNamespace:  false
+  AfterCaseLabel: false
+  AfterClass: false
+  AfterControlStatement: MultiLine
+  AfterEnum: false
+  AfterFunction: true
+  AfterNamespace: false
   AfterObjCDeclaration: false
-  AfterStruct:     false
-  AfterUnion:      false
-  BeforeCatch:     false
-  BeforeElse:      false
-  IndentBraces:    false
+  AfterStruct: false
+  AfterUnion: false
+  AfterExternBlock: false
+  BeforeCatch: false
+  BeforeElse: false
+  BeforeLambdaBody: false
+  BeforeWhile: false
+  IndentBraces: false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: true
+  SplitEmptyNamespace: true
 BreakBeforeBinaryOperators: None
+BreakBeforeConceptDeclarations: true
 BreakBeforeBraces: Custom
+BreakBeforeInheritanceComma: false
+BreakInheritanceList: AfterColon
 BreakBeforeTernaryOperators: true
-BreakConstructorInitializersBeforeComma: true
+BreakConstructorInitializersBeforeComma: false
+BreakConstructorInitializers: AfterColon
 BreakAfterJavaFieldAnnotations: false
-BreakStringLiterals: true
-ColumnLimit:     100
-CommentPragmas:  '^ IWYU pragma:'
+BreakStringLiterals: false
+ColumnLimit: 100
+CommentPragmas: '^ IWYU pragma:'
+CompactNamespaces: false
 ConstructorInitializerAllOnOneLineOrOnePerLine: false
-ConstructorInitializerIndentWidth: 2
+ConstructorInitializerIndentWidth: 4
 ContinuationIndentWidth: 4
 Cpp11BracedListStyle: false
-DerivePointerAlignment: true
-DisableFormat:   false
+DeriveLineEnding: true
+DerivePointerAlignment: false
+DisableFormat: false
+EmptyLineBeforeAccessModifier: Always
 ExperimentalAutoDetectBinPacking: false
-ForEachMacros:
-  - list_for_each
-  - list_for_each_entry
-  - list_for_each_entry_reverse
-  - list_for_each_entry_safe
-  - foreach
-  - Q_FOREACH
-  - BOOST_FOREACH
+FixNamespaceComments: true
+IncludeBlocks: Regroup
 IncludeCategories:
-  - Regex:           '^"(llvm|llvm-c|clang|clang-c)/'
-    Priority:        2
-  - Regex:           '^(<|"(gtest|isl|json)/)'
-    Priority:        3
-  - Regex:           '.*'
-    Priority:        1
-IncludeIsMainRegex: '$'
-IndentCaseLabels: true
-IndentWidth:     4
+ - Regex:           '^[<"]build_config.h'
+   Priority:        1
+ - Regex:           '^[<"]mtf/'
+   Priority:        10
+ - Regex:           '^[<"]mock/'
+   Priority:        11
+ - Regex:           '^[<"]hse/.+/'
+   Priority:        21
+ - Regex:           '^[<"]hse/'
+   Priority:        20
+ - Regex:           '^[<"]tools/'
+   Priority:        30
+ - Regex:           '^"'
+   Priority:        31
+ - Regex:           '^<.*/'
+   Priority:        3
+ - Regex:           '^<'
+   Priority:        2
+ - Regex:           '.'
+   Priority:        100
+# Seems to be no way to disable the "IncludeIsMain" settings,
+# so we set them to patterns we don't expect to see.
+IncludeIsMainRegex: 'XXX'
```

Comment:
Did you try just removing these?

Reply:
Can remove `IncludeIsMainSourceRegex: 'XXX'` but not `IncludeIsMainRegex: 'XXX'`

Reply:
Removed `IncludeIsMainSourceRegex: 'XXX'